errors/abci_test.go

```diff
@@ -156,34 +159,79 @@ func TestABCIInfoHidesStacktrace(t *testing.T) {
 }
 
 func TestRedact(t *testing.T) {
-	if err := Redact(ErrPanic, false); ErrPanic.Is(err) {
-		t.Error("in non-debug mode, redact must not pass through panic error")
+	if err := Redact(ErrPanic); ErrPanic.Is(err) {
+		t.Error("reduct must not pass through panic error")
 	}
-	if err := Redact(ErrPanic, true); !ErrPanic.Is(err) {
-		t.Error("in debug mode, redact should pass through panic error")
-	}
-
-	if err := Redact(ErrNotFound, true); !ErrNotFound.Is(err) {
-		t.Error("in debug mode, redact should pass through weave error")
-	}
-	if err := Redact(ErrNotFound, false); !ErrNotFound.Is(err) {
-		t.Error("in non-debug mode, redact should pass through weave error")
+	if err := Redact(ErrNotFound); !ErrNotFound.Is(err) {
+		t.Error("reduct should pass through weave error")
 	}
 
 	var cerr customErr
-	if err := Redact(cerr, true); err != cerr {
-		t.Error("in debug mode, redact should pass through ABCI code error")
-	}
-	if err := Redact(cerr, false); err != cerr {
-		t.Error("in non-debug mode, redact should pass through ABCI code error")
+	if err := Redact(cerr); err != cerr {
+		t.Error("reduct should pass through ABCI code error")
 	}
 
 	serr := fmt.Errorf("stdlib error")
-	if err := Redact(serr, false); err == serr {
-		t.Error("in non-debug mode, redact must not pass through a stdlib error")
+	if err := Redact(serr); err == serr {
+		t.Error("reduct must not pass through a stdlib error")
+	}
+}
+
+func TestABCIInfoSerializeErr(t *testing.T) {
+	var (
+		// create errors with stacktrace for equal comparision
+		myErrState          = errors.WithStack(ErrState)
```

Comment:
Why use `pkg/errors.WithStack()`, not `weave/errors.Wrap()`, which also adds a stack-trace?

I mean, I am fine for raw stdlib or pkg/errors tests to test non-weave errors, but if you are wrapping something from this module, let's use our Wrap function.

Reply:
👍 good point!